lenskit-eval/src/main/java/org/lenskit/eval/traintest/DataSetBuilder.java

```diff
@@ -21,13 +21,17 @@
 package org.lenskit.eval.traintest;
 
 import com.google.common.base.Preconditions;
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Lists;
+import com.sun.org.apache.bcel.internal.generic.ARRAYLENGTH;
```

Comment:
Stray import - we don't need it, and it comes from `com.sun` which means it's internal to Java and we shouldn't use it.